bin/make_spack

```diff
@@ -49,7 +49,9 @@ bootstrap_patchelf() {
     export pdir
     mkdir $pdir
     PATH="$pdir:$PATH"
-    wget https://spack-cache-1.fnal.gov/binaries/bootstrap/patchelf.linux-scientific7-x86_64
+    curl -OL https://spack-cache-1.fnal.gov/binaries/bootstrap/patchelf.linux-scientific7-x86_64
```

Comment:
I would add `-J --fail` to the curl options and deal with an error condition here.

Reply:
The error code when I run this from CERN is 0 and the file is text.
```
 curl -OJL --fail https://spack-cache-1.fnal.gov/binaries/bootstrap/patchelf.linux-scientific7-x86_64
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   417    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0  1341    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100  9174  100  9174    0     0   4904      0  0:00:01  0:00:01 --:--:--  4904
-bash-4.2$ echo $?
0
```

Reply:
If I omit -L I still get a text file and error code 0
```
  curl -OJ --fail https://spack-cache-1.fnal.gov/binaries/bootstrap/patchelf.linux-scientific7-x86_64
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   417  100   417    0     0    582      0 --:--:-- --:--:-- --:--:--   582
-bash-4.2$ cat patchelf.linux-scientific7-x86_64
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>303 See Other</title>
</head><body>
<h1>See Other</h1>
<p>The answer to your request is located <a href="https://spack-cache-1.fnal.gov/mellon/login?ReturnTo=https%3A%2F%2Fspack%2Dcache%2D1.fnal.gov%2Fbinaries%2Fbootstrap%2Fpatchelf.linux%2Dscientific7%2Dx86_64&amp;IdP=https%3A%2F%2Fidp.fnal.gov%2Fidp%2Fshibboleth">here</a>.</p>
</body></html>
```

Reply:
> The error code when I run this from CERN is 0 and the file is text.
> 
> ```
>  curl -OJL --fail https://spack-cache-1.fnal.gov/binaries/bootstrap/patchelf.linux-scientific7-x86_64
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
>   0   417    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
>   0  1341    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
> 100  9174  100  9174    0     0   4904      0  0:00:01  0:00:01 --:--:--  4904
> -bash-4.2$ echo $?
> 0
> ```

What is the content of the text file here?

Reply:
Much longer with instructions for accessing.

Reply:
Can you pull it back to scisoftbuild01 so I can see, please?

Reply:
Done `~gartung/patchelf.linux-scientific7-x86_64`

Reply:
OK. The following curl command will fail with code 47 fi the requested file is redirected (_e.g._ to the authorization page):
```
curl -OJL --max-redirs 0 --silent --fail https://spack-cache-1.fnal.gov/binaries/bootstrap/patchelf.linux-scientific7-x86_64
```